py/circuitpy_mpconfig.mk

```diff
@@ -222,7 +190,7 @@ endif
 CFLAGS += -DCIRCUITPY_PROTOMATTER=$(CIRCUITPY_PROTOMATTER)
```

Comment:
This is now called `CIRCUITPY_RGBMATRIX`.

Reply:
Was this introduced in the last few days? I remember you mentioning it but I thought I pulled right before submitting this PR...

Reply:
I looked in the latest master, because I thought it was merged already. So maybe try fetch and merge again.